terraform/groups/ecs-service/locals.tf

```diff
@@ -9,7 +9,7 @@ locals {
   docker_repo                = "limited-partnerships-api"
   kms_alias                  = "alias/${var.aws_profile}/environment-services-kms"
   lb_listener_rule_priority  = 23
-  lb_listener_paths          = ["/transactions/*/limited-partnership/*", "/limited-partnership/healthcheck"]
+  lb_listener_paths          = ["/transactions/*/incorporation/*", "/transactions/*/limited-partnership/*", "/limited-partnership/healthcheck"]
```

Comment:
might have been better to have ended the path with limited-partnerships* as i'm not sure if 
/transactions/*/incorporation/*
might redirect other services traffic to ours if their url matches that incorporation pattern.
Not sure how widely that load balancer rule is applied.

Reply:
Agreed. We need to specify `limited-partnership` in the listener patterns. Updated.